src/controller/python/chip/clusters/CHIPClusters.cpp

```diff
@@ -3862,16 +3862,6 @@ chip::ChipError::StorageType chip_ime_AppendCommand_OperationalCredentials_Remov
     cluster.Associate(device, ZCLendpointId);
     return cluster.RemoveFabric(nullptr, nullptr, fabricIndex).AsInteger();
 }
-chip::ChipError::StorageType chip_ime_AppendCommand_OperationalCredentials_RemoveTrustedRootCertificate(
```

Comment:
seems like this was not intended, right?

Reply:
No, this occurred after regen when zap tool made odd modifications.